vitrivr-engine-module-fes/src/main/kotlin/org/vitrivr/engine/base/features/external/implementations/ImageClass.kt

```diff
@@ -0,0 +1,64 @@
+package org.vitrivr.engine.base.features.external.implementations
+
+import org.vitrivr.engine.base.features.external.common.ApiWrapper
+import org.vitrivr.engine.base.features.external.common.ExternalFesAnalyser
+import org.vitrivr.engine.base.features.external.common.FesExtractor
+import org.vitrivr.engine.core.context.IndexContext
+import org.vitrivr.engine.core.context.QueryContext
+import org.vitrivr.engine.core.model.content.element.ContentElement
+import org.vitrivr.engine.core.model.content.element.ImageContent
+import org.vitrivr.engine.core.model.descriptor.struct.LabelDescriptor
+import org.vitrivr.engine.core.model.metamodel.Schema
+import org.vitrivr.engine.core.model.query.Query
+import org.vitrivr.engine.core.model.retrievable.Retrievable
+import org.vitrivr.engine.core.model.retrievable.RetrievableId
+import org.vitrivr.engine.core.model.types.Value
+import org.vitrivr.engine.core.operators.Operator
+import org.vitrivr.engine.core.operators.ingest.Extractor
+import org.vitrivr.engine.core.operators.retrieve.Retriever
+import java.util.*
+
+class ImageClass : ExternalFesAnalyser<ContentElement<*>, LabelDescriptor>() {
```

Comment:
I think the name of this class is misleading. I'd suggest to have `ImageClassification`